modules/cookbooks/pages/kafka_migrator.adoc

```diff
@@ -0,0 +1,633 @@
+= Kafka Migrator
+
+// tag::single-source[]
+
+:description: With Kafka Migrator, you can move your workloads from any Kafka system to Redpanda using a single command. It lets you migrate Kafka messages, schemas, and ACLs quickly and efficiently.
+
```

Comment:
Can we add an intro:

Move your workloads from any Kafka system to Redpanda using a single command. Kafka Migrator lets you migrate Kafka messages, schemas, and ACLs quickly and efficiently.